### PR TITLE
Implement log argument for suggest_int of chainermn

### PR DESCRIPTION
**Pull request by nzw0301**
## Motivation

One of the follow-up PRs related to #1201

## Description of the changes

- Introduce `log` argument for `chainermn` integration.
- Change `low` value to re-use test cases for `suggest_int` for simplicity.

### CODE REVIEW

**Comment by nzw0301** [comment]
@himkt Could you review this PR?

File: tests/integration_tests/test_chainermn.py [hunk 1]
```diff
@@ -395,21 +395,22 @@ def test_suggest_discrete_uniform(storage_mode, comm):
 
     @staticmethod
     @pytest.mark.parametrize("storage_mode", STORAGE_MODES)
-    def test_suggest_int(storage_mode, comm):
-        # type: (str, CommunicatorBase) -> None
+    @pytest.mark.parametrize("enable_log", [False, True])
+    def test_suggest_int(storage_mode, comm, enable_log):
+        # type: (str, CommunicatorBase, bool) -> None
```
Comment (toshihikoyanase) [inline tests/integration_tests/test_chainermn.py:400]: Please use the Python3-style type hinting.
```suggestion
    def test_suggest_int(storage_mode: str, comm: CommunicatorBase, enable_log: bool) -> None:
```

Comment (nzw0301) [reply tests/integration_tests/test_chainermn.py]: I have updated it. Thanks!

**Comment by nzw0301** [comment]
@toshihikoyanase Thank you for your comments! It does not seem to be out of the scope for me. I have included updates of examples you suggested in this PR.

**Comment by toshihikoyanase** [comment]
Let me update the title for the release note.

**Pull request merged**